db/models/categories.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../connection');
+
+exports.fetchCategories = () => {
+    return db.query('SELECT * FROM categories;').then((result) => {
+        console.log(result.rows, "in the model")
```

Comment:
Nooooo, not a console log - please remove this before merging, please remove and commit so I can see before it's merged.